raw/era5_pl_hourly.cfg

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+[parameters]
+client=cds
+dataset=reanalysis-era5-pressure-levels
+# https://confluence.ecmwf.int/display/CKB/Climate+Data+Store+%28CDS%29+API+Keywords
+
+target_path=gs://gcp-public-data-arco-era5/raw/date-variable-pressure_level/{date:%%Y/%%m/%%d}/{variable}/{pressure_level}.nc
+partition_keys=
+    date
+    variable
+    pressure_level
+
+api_url=https://cds.climate.copernicus.eu/api/v2
+
+# go/valentine
+# [parameters.a]
+# api_key=#TODO: put first api key here
+# [parameters.b]
+# api_key=#TODO: put second api key here...
+#
+# See also: https://weather-tools.readthedocs.io/en/latest/Configuration.html#subsections
+
+[selection]
+product_type=reanalysis
+format=netcdf
+
+# See go/ecmwf-grids
+grid=0.25/0.25
+
+date=1959-01-01/to/2023-05-31
+
+# Once an hour for 24 hours, see https://confluence.ecmwf.int/pages/viewpage.action?pageId=118817378
+time=00/to/23
+
+variable=
+    geopotential
+    specific_humidity
+    temperature
+    u_component_of_wind
+    v_component_of_wind
+    vertical_velocity
+    potential_vorticity
+    specific_cloud_ice_water_content
+    specific_cloud_liquid_water_content
+    fraction_of_cloud_cover
+    ozone_mass_mixing_ratio
+
+
+# All 37 pressure levels
+pressure_level=1/2/3/5/7/10/20/30/50/70/100/125/150/175/200/225/250/300/350/400/450/500/550/600/650/700/750/775/800/825/850/875/900/925/950/975/1000
+# Note, the 13 WeatherBench levels are: 50/100/150/200/250/300/400/500/600/700/850/925/1000
+
+
+
+
+
+
```

Comment:
Please remove the empty whitespace.

Reply:
The required modifications have been made.